wicket-jquery-ui-core/src/main/java/com/googlecode/wicket/jquery/core/Options.java

```diff
@@ -138,7 +138,7 @@ public static String fromArray(Object... objects)
 	 */
 	public Options()
 	{
-		this.map = new HashMap<String, Object>();
+		this.map = new HashMap<>();
```

Comment:
I have to avoid using diamonds to get the minimal diffs as possible (with wicket6.x branch, until it is deprecated)